release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -151,6 +152,14 @@ With this release, you can configure an {product-title} cluster on any on-premis
 
 For more information about this feature on bare-metal infrastructure, see xref:../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#nw-osp-services-external-load-balancer_ipi-install-installation-workflow[Services for a user-managed load balancer] in _Setting up the environment for an OpenShift installation_.
 
+[id="ocp-4-16-detect-warning-for-iptables"]
+==== Detect and warning for iptables
+With this release, if you have pods in your cluster using `iptables` rules on either {op-system-base} or third-party software the following event message is given to warn against future deprecation:
```

Comment:
What does `{op-system-base}` expand to? The contrast with "third-party software" here isn't "OpenShift", it's "your own pods".

Basically, what we want to tell users is: if you are using iptables in your own pods, you should stop, and if you're using iptables in someone else's pods, you should tell them to stop.